### PR TITLE
[Fix-4921][UI] Fix download log get return 404

### PR DESCRIPTION
**Pull request by lijufeng2016**
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/development/pull-request.html before opening a pull request.*

## What is the purpose of the pull request

*fix download task log return 404*

### CODE REVIEW

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/log.vue [hunk 1]
```diff
@@ -168,7 +168,7 @@
        * Download log
        */
       _downloadLog () {
-        downloadFile('/log/download-log', {
+        downloadFile('/dolphinscheduler/log/download-log', {
```
Comment (chengshiwen) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/log.vue]: Don't do this, please change to `downloadFile('log/download-log'`

**Review by chengshiwen — CHANGES_REQUESTED** [review]
Please also change `downloadFile('/resources/download', {` to `downloadFile('resources/download', {` in file `dolphinscheduler-ui/src/js/conf/home/pages/resource/pages/file/pages/details/index.vue`

**Review by chengshiwen — APPROVED** [review]
+1

**Review by break60 — APPROVED** [review]
+1

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**